apps/site/lib/site_web/plugs/line_suspensions.ex

```diff
@@ -0,0 +1,46 @@
+defmodule SiteWeb.Plugs.LineSuspensions do
+  @moduledoc """
+    assigns line_suspensions map based on hard-coded values into @suspensions
+    Could potentially add more detail e.g. which stops are served by shuttle,
+    which alternative modes are available, etc.
+  """
+
+  @suspensions %{"Orange" => [~N[2022-08-19T21:00:00], ~N[2022-09-19T04:55:00]]}
```

Comment:
Open to other data structures here, but a benefit of this over, say, `Tuple` is that it's JSON-serializable without extra work. :P